libs/sections/src/header/stories/vl-header.stories-arg.ts

```diff
@@ -15,6 +16,15 @@ export const headerArgs = {
 };
 
 export const headerArgTypes: ArgTypes<typeof headerArgs> = {
+    allowSkeleton: {
```

Comment:
mss `disableSkeleton` & `data-vl-disable-skeleton`

ik denk dat het op zich niet slecht is, dat dit standaard behaviour is maar voor de afnemers die het dan al zelf hebben geïmplementeerd kan dit dan afstaan

wat denken jullie @kspeltix @ksvlaanderen

Reply:
@Goldflow  Bedoel je dat dit voor backwards incompatibility kan zorgen?

Reply:
bedoel dus afweging om:

1. deze functionaliteit standaard te maken en dan moet de afnemer het expliciet afzetten (dit is idd backwards incompatible)
2. te houden zoals het is, dwz als de afnemer het wil, moet hij het zelf aanzetten (geen backwards incompatibiliteit)

maar zou toch voor het 1e optie gaan dus

Reply:
dus zoals afgesproken, blijft zoals het nu is (dus optie 2)